deepmd/pt/model/model/__init__.py

```diff
@@ -182,6 +185,8 @@ def get_standard_model(model_params):
         modelcls = DOSModel
     elif fitting_net["type"] in ["ener", "direct_force_ener"]:
         modelcls = EnergyModel
+    elif fitting_net["type"] in ["prop", "property"]:
```

Comment:
Why two keys here?

Reply:
I think that it can prevent users from writing ‘prop’ by mistake.  I will remove the ‘prop’ option.